package.json

```diff
@@ -24,6 +24,14 @@
                     "type": "string",
                     "default": "",
                     "description": "Configuration file for the code style check"
+                },
+                "haxecheckstyle.sourceFolders": {
```

Comment:
How did it work before, did it check all folders?

Reply:
those folders are used for two things
1. make exlcudes work, because they usually include package / module names but not the source folders, so excludes from your `checkstyle.json` file  didn't properly match, especially in `RELATIVE_TO_SOURCE` mode
2. restricts checkstyle rules to only apply to files below any of the folder names within your workspace. That way you can have libraries / external code insider your workspace folder, that checkstyle won't try to check. Also if you jump to a definition of a haxelib / stdlib type, it will not try to apply your coding style rules.

Reply:
Thanks for clarifying!